autopilot/autopilot.go

```diff
@@ -10,6 +11,11 @@ import (
 	rhpv2 "go.sia.tech/renterd/rhp/v2"
 	"go.sia.tech/renterd/worker"
 	"go.sia.tech/siad/types"
+	"go.uber.org/zap"
+)
+
+const (
+	logFileName = "autopilotlog.json"
```

Comment:
I think `autopilot.log` is fine. There are no rules about what format a `.log` has to be in right?
Also strictly speaking it would be a `.jsonl` file right? Cause it's not a valid json object but instead a json object per line.

Reply:
I hated the name too but was always taught to chose the extension that's most precise. Fair point though about `.jsonl` that slipped my mind. I updated it to `autopilot.log` because it does make for an awful file name 🤷‍♂️